packages/actor-scraper/web-scraper/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+	"extends": "@apify/tsconfig",
+	"compilerOptions": {
+		"outDir": "dist",
+		"module": "ESNext",
```

Comment:
do we really build into esnext? rest of the packages won't be build that way for sure, we need to stick to CJS for now

Reply:
I copied this from the web checker actor I wrote, which runs on ESM, I'll remove this 👍

Reply:
As mentioned in the other comment, let's keep it here for now, those are not regular packages so it might be just fine. I can imagine we will have a separate build process for actors so it might not matter at all.